pandas/core/ops.py

```diff
@@ -1003,8 +1003,18 @@ def _arith_method_SERIES(cls, op, special):
                         if op is divmod else _construct_result)
 
     def na_op(x, y):
-        import pandas.core.computation.expressions as expressions
+        # handle extension array ops
+        # TODO(extension)
+        # the ops *between* non-same-type extension arrays are not
+        # very well defined
+        if (is_extension_array_dtype(x) or is_extension_array_dtype(y)):
+            if (op_name.startswith('__r') and not
+                    is_extension_array_dtype(y) and not
+                    is_scalar(y)):
+                y = x.__class__._from_sequence(y)
+            return op(x, y)
```

Comment:
overlap in comment in #21191 since IntegerArray has an `__init__` that I'm pretty sure can take `left` here (actually below in `wrapper`), I'm hopeful that in this case we can save a lot of trouble by using `dispatch_to_index_op` directly (passing the EA subclass rather than an Index subclass)